Makefile

```diff
@@ -7,9 +7,15 @@ dev:
 test: dev check
 	pytest tests -v -rsXx	
 
-check: dev 
+check:
 	isort -rc -c fonduer/
 	black fonduer/ --check
+	# This is our code-style check. We currently allow the following exceptions:
+	#   - E731: do not assign a lambda expression, use a def
+	#   - W503: line break before binary operator
+	#   - E741: do not use variables named ‘l’, ‘O’, or ‘I’
+	#   - E203: whitespace before ‘:’
+	flake8 fonduer --count --max-line-length=127 --statistics --ignore=E731,W503,E741,E203
```

Comment:
Added flake8 check back into the makefile because it actually makes refactoring a little easier (e.g. highlights modules that are imported and unused or not imported)